zmq.hpp

```diff
@@ -32,20 +32,34 @@
 #endif
 #endif
 
+// included here for _HAS_CXX* macros
+#include <zmq.h>
+
+#if defined(_MSCV_LANG)
```

Comment:
I guess this should be `_MSVC_LANG`? Also in the line below.

Reply:
Yes, good catch.